joomla/joomla-cms
# [sendtest mail] Wrap lines on json error

Pull Request for New Issue.
#### Summary of Changes

Very simple PR to wrap the lines in "Send Mail" button of global configuration when a json error occur.

Before
![image](https://cloud.githubusercontent.com/assets/9630530/15130768/cbf2907a-1642-11e6-8a49-675609aa5ed9.png)

After
![image](https://cloud.githubusercontent.com/assets/9630530/15130774/d6157c16-1642-11e6-9fad-1d56b38cd758.png)
#### Testing Instructions

Code review, or:
1. Apply patch
2. Add in https://github.com/joomla/joomla-cms/blob/staging/administrator/components/com_config/controller/application/sendtestmail.php#L42

``` php
trigger_error('My very big error message. My very big error message. My very big error message. My very big error message. My very big error message. My very big error message. My very big error message. My very big error message. My very big error message. My very big error message. My very big error message. My very big error message. My very big error message. ', E_USER_WARNING);
```
1. Go to global config in the admin panel and send a test mail
2. Check if the error text is wraped
